### PR TITLE
ULP-112 - Template UI changes

### PR DESCRIPTION
**Pull request by Ekta29yadav**

### CODE REVIEW

File: src/app/layouts/doc-view/doc-view.component.scss [hunk 1]
```diff
@@ -15,3 +15,9 @@ footer {
     border-radius: 10rem !important;
     background-color: #fff;
 }
+.flex-justify-center{
+    justify-content: center;
+}
+.flloat-left{
```
Comment (itsvick) [inline src/app/layouts/doc-view/doc-view.component.scss:21]: @Ekta29yadav here is typo. Please fix

File: src/app/layouts/doc-view/doc-view.component.ts [hunk 2]
```diff
@@ -73,7 +73,7 @@ export class DocViewComponent implements OnInit {
         //     this.getPDF(res?.schema, res.template?.template);
         // });
 
-        this.getTemplate().subscribe((res) => {//clf16wnze0002tj14mv1smo1w
+        this.getTemplate('clf16wnze0002tj14mv1smo1w').subscribe((res) => {//clf16wnze0002tj14mv1smo1w
```
Comment (itsvick) [inline src/app/layouts/doc-view/doc-view.component.ts:76]: @Ekta29yadav please dont commit this id for now

File: src/templates/certificate.html [hunk 3]
```diff
@@ -1,90 +1,88 @@
...
+        </style>
+    </head>
+    <body>
+        <div class='body'>
+            <div class='center'>
+                <div class='text-center'><div class='font-600 fs-20 py-20'>{{studentName}}</div></div>
+                <div class='d-flex jc-sp-bw'>
+                    <div class='w-45'>
+                        Student ID
+                        <div class='font-600'>{{studentID}}</div>
```
Comment (itsvick) [inline src/templates/certificate.html:62]: @Ekta29yadav Can u make this `studentID` as `studentId` same being used in schema

File: src/templates/certificate.html [hunk 4]
```diff
@@ -1,90 +1,88 @@
...
+                    </div>
+                    <div class='w-45'>
+                        Gaurdian Name
+                        <div class='font-600'>{{guardianName}}</div>
+                    </div>
+                    <div class='w-45'>
+                        School ID
+                        <div class='font-600'>{{schoolID}}</div>
```
Comment (itsvick) [inline src/templates/certificate.html:70]: @Ekta29yadav same here

File: src/app/layouts/doc-view/doc-view.component.scss (hunk 1 above)
Comment (Ekta29yadav) [reply src/app/layouts/doc-view/doc-view.component.scss]: done

File: src/app/layouts/doc-view/doc-view.component.ts (hunk 2 above)
Comment (Ekta29yadav) [reply src/app/layouts/doc-view/doc-view.component.ts]: done

File: src/templates/certificate.html (hunk 3 above)
Comment (Ekta29yadav) [reply src/templates/certificate.html]: done

File: src/templates/certificate.html (hunk 4 above)
Comment (Ekta29yadav) [reply src/templates/certificate.html]: done

**Pull request merged**